paddle/contrib/inference/demo/CMakeLists.txt

```diff
@@ -14,3 +14,40 @@
 #
 
 inference_api_test(simple_on_word2vec ARGS test_word2vec)
+
+set(DEMO_INSTALL_DIR "${THIRD_PARTY_PATH}/inference_demo")
+set(mobilenet_url http://paddlemodels.bj.bcebos.com/inference-vis-demos%2Fmobilenet.tar.gz)
```

Comment:
no need to set a variable for url, just pass it as the argument of  `inference_download_test_demo`

Reply:
Got it